src/js/controller/DrawingController.js

```diff
@@ -140,6 +142,12 @@
     $.publish(Events.MOUSE_EVENT, [event, this]);
     var frame = this.piskelController.getCurrentFrame();
     var coords = this.getSpriteCoordinates(event.clientX, event.clientY);
+    if (typeof event.targetTouches != 'undefined' &&
+        typeof event.targetTouches[0] != 'undefined' &&
+        typeof event.targetTouches[0].pageX != 'undefined' &&
+        typeof event.targetTouches[0].pageY != 'undefined') {
```

Comment:
Might be enough to simply check if the objects are not falsy ````if(event.targetTouches && event.targetTouches[0] ...)```` and skip the checks on pageX and pageY, since targetTouches contains Touch instances.